x/bep3/client/cli/tx.go

```diff
@@ -54,36 +59,50 @@ func GetCmdCreateAtomicSwap(cdc *codec.Codec) *cobra.Command {
 			recipientOtherChain := args[1] // same as OtherExecutor.DeputyAddress
 			senderOtherChain := args[2]
 
-			randomNumberHash, err := types.HexToBytes(args[3])
-			if err != nil {
-				return err
+			// Timestamp defaults to time.Now() unless it's explicitly set
+			var timestamp int64
+			if strings.Compare(args[3], "now") == 0 {
+				timestamp = time.Now().Unix()
```

Comment:
```suggestion
				timestamp = tmtime.Now().Unix()
```

Use tendermint's Now method for time.Now

Reply:
https://github.com/Kava-Labs/kava/pull/388/commits/5c4832e2209b183469c84c9b3fa5218114da6b75